python/sauron-core/core/analysis/stats.py

```
@@ -0,0 +1,197 @@
+                                1.0 / (1.0 + np.exp(-5 * (np.mean(finite_values) - 0.3)))
+                            )
+                        else:
+                            metrics[f'{feature}_mean'] = 0.0
+                            metrics[f'{feature}_ai_score'] = 0.5
+
+        except Exception as e:
+            print(f"Error in gradient pattern analysis: {str(e)}")
+
+        return metrics
+
+    def _compute_ai_probability(self, metrics: Dict[str, float]) -> float:
```

Ruff (0.8.2)

Use `dict` instead of `Dict` for type annotation

Replace with `dict`

(UP006)